js/core/renderer.d.ts

```diff
@@ -0,0 +1,256 @@
+interface JQueryCoordinates {
```

Comment:
I suggest renaming all those `jQueryXXX` types to `jQueryLikeXXX` e.g.:
```suggestion
interface JQueryLikeCoordinates {
```

Maybe we should move all of them to a separate folder.

Reply:
I'd prefer to avoid `jquery` word usage everywhere except the `integrations/jquery` folder.

Reply:
Leave part of the name XXX